packages/squidex/src/rest.ts

```diff
@@ -20,18 +20,25 @@ export interface Filter {
     | 'ne'
     | 'contains'
     | 'startsWith'
-    | 'endsWith';
-  value: string[] | number[] | string | number | boolean;
+    | 'endsWith'
+    | 'empty';
+  value: string[] | number[] | string | number | boolean | null;
+}
+
+export interface LogicalFilter {
+  and?: Filter[];
+  not?: Filter;
+  or?: Filter[];
```

Comment:
I've made this query in the Squidex UI

<img width="1408" alt="Screen Shot 2022-09-21 at 06 19 44" src="https://user-images.githubusercontent.com/16595804/191467332-4aa278e1-8ffa-4a8b-b9d3-8783fac175f7.png">

that generated this query in the network tab. So I could see that this `and` and `or` blocks can be nested :)
<img width="658" alt="Screen Shot 2022-09-21 at 06 21 27" src="https://user-images.githubusercontent.com/16595804/191467406-42326042-b7e4-4eef-a5e3-6872655efb09.png">